src/lib/requests/mojaloopRequests.js

```diff
@@ -22,26 +22,37 @@ const { ResponseType } = require('./common');
 class MojaloopRequests extends BaseRequests {
     constructor(config) {
         super(config);
+
+        this._config = config;
     }
 
     /**
      * Executes a GET /parties request for the specified identifier type and identifier
      *
      * @returns {object} - JSON response body if one was received
      */
-    async getParties(idType, idValue, idSubValue, destFspId) {
+    async getParties(idType, idValue, idSubValue, destFspId, headers = {}) {
+        if (headers.tracestate) {
+            const TRACESTATE_KEY_CALLBACK_START_TS = 'tx_callback_start_ts';
+            headers.tracestate = headers.tracestate + `,${TRACESTATE_KEY_CALLBACK_START_TS}=${Date.now()}`;
+        }
         const url = `parties/${idType}/${idValue}`
             + (idSubValue ? `/${idSubValue}` : '');
-        return this._get(url, 'parties', destFspId);
+        return this._get(url, 'parties', destFspId, headers);
     }
 
     /**
      * Executes a PUT /parties request for the specified identifier type and indentifier
      */
-    async putParties(idType, idValue, idSubValue, body, destFspId) {
+    async putParties(idType, idValue, idSubValue, body, destFspId, headers = {}) {
```

Comment:
I think these headers are really specific to perfomance testing.
Any reason as to why you don't just specify the headers in the sdk instead of doing conditional logic here?